common/djangoapps/third_party_auth/models.py

```diff
@@ -140,6 +140,17 @@ class ProviderConfig(ConfigurationModel):
             "authentication using the correct link is still possible."
         ),
     )
+    max_session_length = models.PositiveIntegerField(
+        null=True,
+        blank=True,
+        default=None,
+        verbose_name='Max session length (seconds)',
+        help_text=_(
+            "If this option is set, then users logging in using this SSO provider will have "
```

Comment:
It might be worth noting explicitly what happens if this is set to None. From reading the code it seems to be falling back on the platform's default behavior, but it's always better to be explicit.